<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.141955, y1:0.233, x2:0.756, y2:0.880682, stop:0 rgba(84, 162, 165, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(241, 241, 241, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>150</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>350</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Enter1">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">Enter1 {
    color: #333;
    
    border-radius: 20px;
    border-style: outset;
    
	background-color: qradialgradient(spread:pad, cx:0.37, cy:0.511364, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(84, 68, 178, 255), stop:1 rgba(255, 255, 255, 255));
    padding: 5px;
    }

Enter1:hover {
    background-color: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

Enter1:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Enter4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>500</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">Enter4 {
    background-color: orange;
border-style: outset;
border-width: 2px;
border-radius: 15px;
border-color: black;
padding: 4px;
    }

Enter4:hover {
    background-color: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

Enter4:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }</string>
     </property>
     <property name="text">
      <string>RUN</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Enter The working directory</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Features to select From</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Input Country Layer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Input Shops Layer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Choose Country</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Select Type of Shop</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Enter Name of Output</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>400</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>450</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(158, 158, 158, 255), stop:1 rgba(255, 255, 255, 255));
selection-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 10pt &quot;Lucida Bright&quot;;
background-color: rgb(255, 255, 255,0);</string>
     </property>
     <property name="text">
      <string>Enter Output Directory</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
